### ISSUE TITLE
Arbtt does not build on nixpkgs due to old Setup.hs

### ISSUE THREAD

**Issue by markus1189** [nomeata/arbtt#43]
Hey,

unfortunately even after #42 `arbtt` does not yet build on nixpkgs.  It's due to an old function being used in `Setup.hs`:

```haskell
[1 of 1] Compiling Main             ( Setup.hs, /build/Main.o )

Setup.hs:26:17: error:
    Variable not in scope:
      rawSystemProgram
        :: Distribution.Verbosity.Verbosity
           -> ConfiguredProgram -> [[Char]] -> IO a0
   |
26 |                 rawSystemProgram verb configuredProg
   |                 ^^^^^^^^^^^^^^^^
```

This was actually already fixed in 8270c3be6f4cbd313eb71367f46a0ba29e7990c5, but I guess we would need a new hackage release, as the current one is from `2019-03-16T17:26:24Z`

**Comment by nomeata** [comment]
Just uploaded 0.10.4. Does that work for you?

**Comment by markus1189** [comment]
Worked fine locally :tada: Once nixpkgs has the latest hackage update I will fix it there, too.

Thanks for the quick response!
